app/app.go

```diff
@@ -43,6 +40,9 @@ import (
 	"github.com/sygmaprotocol/sygma-core/chains/evm/listener"
 	"github.com/sygmaprotocol/sygma-core/chains/evm/transactor/monitored"
 	"github.com/sygmaprotocol/sygma-core/chains/evm/transactor/transaction"
+	"github.com/sygmaprotocol/sygma-core/chains/substrate/client"
+	"github.com/sygmaprotocol/sygma-core/chains/substrate/connection"
+	core_substrate_listener "github.com/sygmaprotocol/sygma-core/chains/substrate/listener"
```

Comment:
I think generally everywhere else we use camel case.

Reply:
Yes, I've made it part of the next PR already :)